### PR TITLE
docs(faq): clarify browser support

### PR DESCRIPTION
**Pull request by Narretz**
The info about supported browsers is taking from the Travis tasks.

/cc @petebacondarwin

### CODE REVIEW

File: docs/content/misc/faq.ngdoc [hunk 1]
```diff
@@ -86,9 +86,15 @@ Yes. See instructions in {@link downloading}.
...
-Explorer Compatibility} for more details on supporting legacy IE browsers.
+We run our extensive test suite against the following browsers: the latest versions of Chrome,
+Firefox, Safari, and Safari for iOs, as well as Internet Explorer versions 9-11. See {@link guide/ie
+Internet Explorer Compatibility} for more details on supporting legacy IE browsers.
+
+If a browser is untested, it doesn't mean it won't work; for example, older Android (2.3.x)
+is supported in the sense that we avoid the dot notation for reserved words as property names,
+but we don't actively fix bugs against it. You can also expect browsers to work that share a large
+part of their codebase with a browser we test, such as Opera > version 12 (uses the Blink engine),
+or the various Firefox derivates.
```
Comment (gkalpak) [inline docs/content/misc/faq.ngdoc]: _derivates_  -->  _derivatives_ ?

File: docs/content/misc/faq.ngdoc [hunk 2]
```diff
@@ -86,9 +86,15 @@ Yes. See instructions in {@link downloading}.
...
-We run our extensive test suite against the following browsers: Safari, Chrome, Firefox, Opera 15,
-IE9 and mobile browsers (Android, Chrome Mobile, iOS Safari). See {@link guide/ie Internet
-Explorer Compatibility} for more details on supporting legacy IE browsers.
+We run our extensive test suite against the following browsers: the latest versions of Chrome,
+Firefox, Safari, and Safari for iOs, as well as Internet Explorer versions 9-11. See {@link guide/ie
+Internet Explorer Compatibility} for more details on supporting legacy IE browsers.
+
+If a browser is untested, it doesn't mean it won't work; for example, older Android (2.3.x)
+is supported in the sense that we avoid the dot notation for reserved words as property names,
+but we don't actively fix bugs against it. You can also expect browsers to work that share a large
```
Comment (petebacondarwin) [inline docs/content/misc/faq.ngdoc]: `but we don't actively fix bugs against it` should really be `but we don't actively test changes against it`

File: docs/content/misc/faq.ngdoc (hunk 1 above)
Comment (petebacondarwin) [inline docs/content/misc/faq.ngdoc]: +1

**Comment by petebacondarwin** [comment]
Two small changes. Otherwise ready to merge. Good job @Narretz

File: docs/content/misc/faq.ngdoc (hunk 1 above)
Comment (Narretz) [inline docs/content/misc/faq.ngdoc]: I blame German ;p

**Comment by petebacondarwin** [comment]
:thumbsup:

**Pull request closed without merging**